src/app/modules/cart/cart.component.ts

```diff
@@ -7,9 +10,76 @@ import { Component, OnInit } from '@angular/core';
 })
 export class CartComponent implements OnInit {
 
-  constructor() { }
+  products: CartItem[];
+
+  constructor(private _cartService: CartService, private router: Router, public user: AuthService) { 
+    this.products = _cartService.getCartItems() || [];
+  }
 
   ngOnInit() {
   }
 
+  totalPriceOfProducts() {
+    let totalPrice = 0;
+    for(let i = 0; i < this.products.length; i++) {
+      totalPrice += this.products[i].product_price * this.products[i].product_quantity;
+    }
+    return totalPrice;
+  }
+
+  getProductSummary(price: number, quantity: number) {
```

Comment:
public/private getProductSummary(price: number, quantity: number): number {

add private or public to all functions and what type they return